extensions/cornerstone/src/init.tsx

```diff
@@ -371,10 +392,10 @@ export default async function init({
   }
 
   // eventTarget.addEventListener(EVENTS.ELEMENT_ENABLED, elementEnabledHandler);
-  eventTarget.addEventListener(EVENTS.ELEMENT_ENABLED, elementEnabledHandler.bind(null));
+  // eventTarget.addEventListener(EVENTS.ELEMENT_ENABLED, elementEnabledHandler.bind(null));
```

Comment:
could you pls explain?

Reply:
We don't use those anymore, they were from wen's change that we reverted